akka-cluster/src/main/scala/akka/cluster/protobuf/ClusterMessageSerializer.scala

```diff
@@ -166,8 +166,11 @@ class ClusterMessageSerializer(val system: ExtendedActorSystem) extends BaseSeri
     builder.setAllowLocalRoutees(settings.allowLocalRoutees)
       .setMaxInstancesPerNode(settings.maxInstancesPerNode)
       .setTotalInstances(settings.totalInstances)
+      .addAllUseRoles(settings.useRoles.asJava)
 
+    // for backwards compatibility
     settings.useRole.foreach(builder.setUseRole)
+
```

Comment:
This means that if using rolling upgrades and a new node has ["a", "b"] and sends to an old node, that will only require the "a" role, no way around that but we should probably document that if using rolling upgrades you should move to the latest versions of Akka first, and then start using multiple roles in a separate rolling upgrade, in the migration guide (I think it would go in `migration-guide-2.4.x-2.5.x.md`)